src/uct/api/uct.h

```diff
@@ -318,6 +318,7 @@ typedef struct uct_tl_resource_desc {
     char                     tl_name[UCT_TL_NAME_MAX];   /**< Transport name */
     char                     dev_name[UCT_DEVICE_NAME_MAX]; /**< Hardware device name */
     uct_device_type_t        dev_type;     /**< Device type. To which UCT group it belongs to */
+    ucs_sys_device_t         sys_device;   /**< Index that ties the device to a bus_id */
```

Comment:
is bus id is a array ? if so where ? It is not very clear to what object the index points.

Reply:
@shamisp Agreed. The documentation doesn't point to what bus_id is. bus_id refers to `ucs_sys_bus_id_t` struct and I've pointed to that in the description now.